internal/operands/metrics/reconcile.go

```diff
@@ -59,6 +67,47 @@ const (
 	operandComponent = common.AppComponentMonitoring
 )
 
+func reconcilePrometheusMonitor(request *common.Request) (common.ReconcileResult, error) {
+	return common.CreateOrUpdate(request).
+		ClusterResource(newServiceMonitorCR(request.Namespace)).
```

Comment:
Can this be `NamespacedResource()`? It is in the same namespace as SSP CR.
Then garbage collector will work for them, and you don't need to add them to `Cleanup()` method.

Reply:
done.